src/DataObject/GridColumnConfig/Value/AbstractValue.php

```diff
@@ -23,9 +23,9 @@ abstract class AbstractValue implements ValueInterface
 
     protected string $label;
 
-    protected mixed $context = null;
+    protected mixed $context;
 
-    public function __construct(\stdClass $config, mixed $context = null)
+    public function __construct(\stdClass $config, mixed $context)
```

Comment:
@blankse For me this looks like a breaking change. Is there a special reason why do you changed it?

Reply:
@blankse That would be great if you create a setLocaleService() method, that we don't change the `AbstractValue`. Thanks

Reply:
@robertSt7 Done: https://github.com/pimcore/admin-ui-classic-bundle/pull/339/commits/e1ae19c5c2e9c9c12ad19cc81ec2eb58c408f324